php/class-ad-layers-widget.php

```diff
@@ -64,7 +64,10 @@ public function form( $instance ) {
 			<p>
 				<label for="<?php echo esc_attr( $this->get_field_id( 'ad_unit' ) ) ?>"><?php esc_html_e( 'Ad Unit', 'ad-layers' ) ?></label>
 				<br />
-				<?php echo $this->ad_unit_select_field_html( $ad_unit ) ?>
+				<?php
+				// @TODO Add and test escaping.
```

Comment:
Just a ping to make sure this TODO is intended for future work and not something that should be included here.

Reply:
yep! this is intended for future clean up.